docs/design/manipulation/analysis.md

```diff
@@ -0,0 +1,195 @@
+  multibody dynamics engine, a “systems framework” for organizing and combining
+  system models from a library into a block diagram, and a optimization
+  framework for mathematical programming.
+  Drake-ROS is mostly geared towards someone who is doing most things in Drake
+  (like motion planning) but wants to connect to ROS for either commands or
+  visualization, no integration with ros_control or Moveit.
+
+* [IssacSim](https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/tutorial_advanced_adding_new_manipulator.html)
+  NVIDIA simulator using NVIDIA PhysX framework. It has its own sub-product
+  called [Cortex](https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/tutorial_cortex_overview.html)
+  for implementing the whole mmanipulation pipeline from detection to control.
+  It has many GUI tools to help with manipulation tuning and design.
+
+* [Simulink](https://www.mathworks.com/solutions/robotics/robot-manipulators.html) - Mathworks.
+  Mathworks offers a large collection of products to address all the aspect of
+  the manipulation workflow: Use functionalities for inverse/forward kinematics
+  and dynamics, motion plan, trajectory generation, and collision checking
+
+## Design considerations
+
+### Implementation in O3DE
+
+### Use of MoveIt 2
+
+[MoveIt 2](https://moveit.picknik.ai/humble/index.html) is the main motion
+planner framework used by ROS 2. There are many ways of interacting with it. In
+this design, the `ROS2ManipulationRequester` component should act as the
+interface for accepting inputs in the form of request for manipulation and call
+MoveIt 2 ([see the code needed to implement a Pose for the end
+effector](https://moveit.picknik.ai/humble/doc/tutorials/your_first_project/your_first_project.html))
+or any other motion planner.
+
+The `ROS2ManipulationRequester` component should also take care of controlling
+the status of the request and provide internal O3DE Gem information on the
+status of the execution.
+
+### Use of controllers: ros2_control and native O3DE
+
+The integration of the [ros2_control](https://github.com/ros-controls/ros2_control) into the
+architecture have some difficulties:
+
+- controllers are typically not modular in a sense that they handle everything
+ (parameters, communication and computations).
+ - we would like to handle some of these elements separately for more
+   flexibility and better UX.
+- some of their design is not as flexible (likely since they have been only
+ used with Gazebo so far).
+ - for example, we would like to handle parameters in the Editor, without
+   running publishers/subscribers already.
+
+The benefits are important though. It is valuable to reuse implementations
+which are tried and tested, and benefit from the joint effort of the community.
+There should be the option of using native O3DE controllers if there is a good
+reason to do so like big performance improvements or much better integration in
+the UI.
```

Comment:
Similar to above, I am strongly in favor of migrating this section to `architecture.md`

```suggestion
```